src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -15,6 +15,7 @@
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
   <ItemGroup>
+    <Reference Include="Microsoft.VisualBasic" />
```

Comment:
oh, i see, for CSV parsing. had no idea. i wonder how VB's apparently general purpose type handles enquoting of strings that contain the delimiter (a comma). Does it have some special understanding of this scenario? Anyway, no real note here.

---
In reply to: [90078642](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90078642) [](ancestors = 90078642)